helm/alfresco-content-services/community_values.yaml

```diff
@@ -29,6 +29,7 @@ share:
   replicaCount: 1
   image:
     repository: alfresco/alfresco-share
+    tag: 7.3.0
```

Comment:
new PR because updatecli always compare against master so this is not really fixing anything here
https://github.com/Alfresco/acs-deployment/pull/875